### PR TITLE
[WIP] Another SSL Providers fix

### PR DESCRIPTION
**Pull request by pavelzag**
Purpose or Intent

Fixing another issue that occures when adding the Hawkular tab validation

### CODE REVIEW

File: cfme/common/provider.py [hunk 1]
```diff
@@ -107,8 +107,11 @@ def form(self):
                 ],
 
                 "Hawkular": [
-                    ('hawkular_validate_btn', form_buttons.validate),
-                ]
+                    ('hawkular_validate_btn', {
+                        version.LOWEST: form_buttons.validate,
+                        '5.6': None
+                    }),
+                ],
```
Comment (jkrocil) [inline cfme/common/provider.py]: No, not like this. If it's not there, just don't pass data into it.

Comment (pavelzag) [inline cfme/common/provider.py]: Can you elaborate? Not sure I understood..

Comment (jkrocil) [inline cfme/common/provider.py]: https://github.com/ManageIQ/integration_tests/blob/master/cfme/common/provider.py#L759

Comment (jkrocil) [inline cfme/common/provider.py]: That's where the version check should be, IIUC.

Comment (pavelzag) [inline cfme/common/provider.py]: Gotcha, checking

**Comment by pavelzag** [comment]
@jkrocil Please have a look, Jan.

File: cfme/common/provider.py [hunk 2]
```diff
@@ -756,7 +756,7 @@ def _fill_credential(form, cred, validate=None):
                 'token_verify_secret': cred.verify_token,
                 'validate_btn': validate
             })
-            if validate:
+            if validate_hawkular:
```
Comment (izapolsk) [inline cfme/common/provider.py]: @pavelzag, could you please show me where you want to use this new option ?
btw, could you please explain what is the reason for it ?

Comment (pavelzag) [inline cfme/common/provider.py]: @izapolsk Per Jan's request I included the changes in the test that will use this path:
https://github.com/ManageIQ/integration_tests/pull/4405

Comment (izapolsk) [inline cfme/common/provider.py]: @pavelzag , I suppose this PR can be closed then, right ?

Comment (pavelzag) [inline cfme/common/provider.py]: Yes, can be closed

**Comment by pavelzag** [comment]
Doubles this:
https://github.com/ManageIQ/integration_tests/pull/4405

**Pull request closed without merging**